playwright/tests/page-objects/control-bar.ts

```diff
@@ -18,7 +18,7 @@ export class ControlBar {
   }
 
   async toggleBrowser() {
-    const button = this.bar.getByRole('switch', { name: 'Simulate' });
+    const button = this.bar.getByLabel('Open Simulation (⇧+S)').getByRole('button');
```

Comment:
getByRole('button', {name: 'Open Simulation'});
shortcut should not be needed in name